react-app/src/styles/components/markdown.scss

```diff
@@ -0,0 +1,8 @@
+/* stylelint-disable */
+.w-md-editor-preview {
```

Comment:
can we move this file to `src/components/MarkdownEditor/` as `MarkdownEditor.module.scss`

and only import at `MarkdownEditor.tsx` instead of global?

Reply:
there's also `MDEditor.Markdown`, maybe we import and export it in `MarkdownEditor.tsx`

Reply:
Importing at `MarkdownEditor.tsx` doesn't seem to be working

Reply:
did you add `:global`? 🤔